Makefile

```diff
@@ -86,16 +86,25 @@ revive: scripts/go/bin/revive
 
 # create docker-compose file with provided sources and start them
 # example: make devenv sources=postgres,openldap
+ifeq ($(sources),)
+devenv:
+	@printf 'You have to define sources for this command \nexample: make devenv sources=postgres,openldap\n'
+else
 devenv: devenv-down
 	$(eval targets := $(shell echo '$(sources)' | tr "," " "))
 
 	@cd devenv; \
-	./create_docker_compose.sh $(targets); \
+	./create_docker_compose.sh $(targets) && rm -f devenv/docker-compose.yaml
```

Comment:
This seems weird to me. Doesn't this mean the file that was just created will be deleted after?

Reply:
Oooh, that's not right, should not have coded while tired :)

Reply:
Oh damn, it's actually right, haha.

So idea is if `create_docker_compose.sh` failed it still means it created compose file.
Like
```bash
markelog at demi in ~/Workspace/gocode/src/github.com/grafana/grafana/devenv
$ ./create_docker_compose.sh openldap,postgress
Deleting docker-compose.yaml
Adding Compose header to docker-compose.yaml
docker/blocks/openldap,postgress is not a directory

$ cat docker-compose.yaml
version: "2"
services:
```
(postgress should be with one `s` at the end). Incorrect `docker-compose.yaml` is still there